tools/config-generator/types.go

```diff
@@ -1,5 +1,19 @@
 package main
```

Comment:
Copyright header is missed for this file.

Reply:
done